cmd/govim/main.go

```diff
@@ -456,3 +467,12 @@ func (g *govimplugin) defineHighlights() {
 	}
 	g.vimstate.BatchEnd()
 }
+
+func getEnvVal(env []string, varname string) (string, bool) {
```

Comment:
Nits, is the bool really needed? It isn't used at all in this PR

Reply:
Thanks, good catch. I forgot to refactor this.